app/models/financialDetails/ChargeItem.scala

```diff
@@ -69,6 +69,12 @@ case class ChargeItem (
 
   def getDueDate: LocalDate = dueDate.getOrElse(throw MissingFieldException("documentDueDate"))
 
+  def getDisplayDueDate: LocalDate = if (isLatePaymentInterest && isPaid) {
```

Comment:
:thinking: - we would need to discuss how we are going to extend ChargeItem type going forward

Reply:
Just moving logic from the view into the model level

Reply:
* I would suspect ChargeItem would become yet another place where we put lots of UI / view specific logic;
we might need to review our design approach: ie wrap ChargeItem into another viewModel/case class or design individual TransactionItem per each page, not sure might worth to raise investigate|Spike story?